core/src/main/java/hera/core/commands/ToggleCommand.java

```diff
@@ -0,0 +1,46 @@
+package hera.core.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.Guild;
+import discord4j.core.object.entity.Member;
+import discord4j.core.object.entity.MessageChannel;
+import hera.core.HeraUtil;
+import hera.database.entities.mapped.Command;
+import hera.database.entities.mapped.GuildSettings;
+import hera.database.entities.mapped.Localisation;
+import hera.database.entities.mapped.ModuleSettings;
+import hera.database.types.GuildSettingKey;
+import hera.database.types.LocalisationKey;
+import reactor.core.publisher.Mono;
+
+import java.awt.*;
+import java.util.List;
+
+import static hera.store.DataStore.STORE;
+
+public class ToggleCommand {
+	public static Mono<Void> execute(MessageCreateEvent event, Guild guild, Member member, MessageChannel channel, List<String> params) {
+		Command cmd = STORE.commands().forName(params.get(0)).get(0);
```

Comment:
Command could be null here. Instead of `.get(0)` get the List from STORE and check if it is empty.

Maybe return a message when the command does not exist